src/software/thunderscope/robot_diagnostics/robot_view.py

```diff
@@ -8,21 +8,58 @@
 from software.thunderscope.thread_safe_buffer import ThreadSafeBuffer
 
 
-class RobotView(QScrollArea):
-    """Class to show a snapshot of the robot's current state.
+class RobotViewComponent(QWidget):
```

Comment:
Is it possible to make the widgets a bit closer to each other? There seems to be a lot of padding/margin between them, making them take more space than required

Reply:
I'll see how to reduce the padding, haven't worked much with styling stuff before